lib/dockerregistry/types.go

```diff
@@ -154,6 +154,14 @@ type Manifest struct {
 	// registries, in which case we would have more than just Src/Dest.
 	Registries []RegistryContext
 	Images     []Image `yaml:"images,omitempty"`
+
+	// Filestores contains the source and destination (Src/Dest) filestores.
+	// Filestores are (for example) GCS or S3 buckets.
+	// It is possible that in the future, we support promoting to multiple
+	// filestores, in which case we would have more than just Src/Dest.
+	Filestores []*Filestore `yaml:"filestores,omitempty"`
```

Comment:
Mostly just paranoid habit.  If we don't use pointers, it's all too easy to (shallow) copy the object on every loop iteration.  We should switch to something like this:

```
for i := range o.Filestores {
file := &o.Filestores[i]
}
```

That said, the tradeoff is less efficient memory packing, so I'll switch to the packed struct.  Also because I don't think this is performance critical code, so a few extra copies won't really matter!